app/models/studio_image.rb

```diff
@@ -5,19 +5,4 @@ def self.paths(studio)
       [kk, path]
     end.to_h
   end
-
-  def self.get_path(studio, size = 'medium')
```

Comment:
the thing that was calling this is doing a lot of the same magic and we don't use `studio data` anymore so this method becomes pretty stupid.